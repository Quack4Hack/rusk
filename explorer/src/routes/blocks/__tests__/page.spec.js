import { describe, expect, it } from "vitest";
import { render } from "@testing-library/svelte";
import Blocks from "../+page.svelte";

describe("Blocks", () => {
  it("should render the Blocks page", () => {
    const { container } = render(Blocks, {});

    expect(container.firstChild).toMatchSnapshot();
  });
});
