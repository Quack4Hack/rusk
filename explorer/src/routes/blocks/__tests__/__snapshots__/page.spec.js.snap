// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Blocks > should render the Blocks page 1`] = `
<div>
  <div>
    All Blocks
  </div>
  <!--&lt;+page&gt;-->
</div>
`;
